lib/rubocop/cop/minitest/assert_respond_to.rb

```diff
@@ -38,13 +37,20 @@ def autocorrect(node)
           lambda do |corrector|
             assert_with_respond_to(node) do |_, object, method, rest_args|
               custom_message = rest_args.first
-              preferred = [object, method, custom_message]
-                          .compact.map(&:source).join(', ')
+              preferred = build_preferred_arguments(object, method, custom_message)
               replacement = "assert_respond_to(#{preferred})"
               corrector.replace(node.loc.expression, replacement)
             end
           end
         end
+
+        private
+
+        def build_preferred_arguments(receiver, method, message)
```

Comment:
@koic Since we are allowing `assert respond_to? :some_method`. Shouldn't we add the change in the example here: https://github.com/rubocop-hq/rubocop-minitest/pull/40/files#diff-8a61a4905f40fa4acda025d5ed4bbb66R9

WDYT?

Reply:
It seems that `assert(respond_to?(:some_method))` does not exist in the examples and documentation. What does that mean?

Reply:
I wanted to ask if we should update the example and documentation as follows

```
# Good
assert_respond_to(self, some_method)

# Bad
assert(respond_to?(:some_method))
```

Reply:
I get it! I updated the examples. Thanks for your explanation!